src/components/App.js

```diff
@@ -0,0 +1,44 @@
+import ButtonPannel from './ButtonPanel';
+import Display from './Display';
+import Button from './Button';
+
+function App() {
+  const buttons = [
+    'AC',
+    'X²',
+    '%',
+    '/',
+    '9',
+    '8',
+    '7',
+    '*',
+    '6',
+    '5',
+    '4',
+    '+',
+    '3',
+    '2',
+    '1',
+    '-',
+    '0',
+    '.',
+    '+/-',
+    '=',
+  ];
+  return (
+    <>
+      <div className="App">
+        <div className="Appcalculator">
+          <Display />
+          <ButtonPannel>
+            {buttons.map(button => (
+              <Button key={button} name={button} />
+            ))}
+          </ButtonPannel>
```

Comment:
Good job so far 👍 ....

But we need to move this mapping implementation inside the `buttonPanel.js`
![image](https://user-images.githubusercontent.com/31897434/126005548-6e5b9231-dfb1-4f98-a103-48ad37e3800e.png)


![image](https://user-images.githubusercontent.com/31897434/126005577-12ea9b50-8269-4930-8fe9-5ffb3d83a94b.png)